tracpro/polls/migrations/0030_reset_question_types.py

```diff
@@ -21,7 +20,8 @@ def guess_question_type_from_rules(apps, schema_editor):
     for org in apps.get_model('orgs', 'Org').objects.all():
         client = get_temba_client(org)
         for poll in org.polls.all():
-            rulesets = client.get_flow_definition(poll.flow_uuid).rule_sets
+            exports = client.get_definitions(flows=[poll.flow_uuid])
+            rulesets = exports.flows[0].rule_sets
```

Comment:
This would be a later optimization, but it looks like the v2 API would let us fetch the rulesets for all the polls at once (by passing all the flow uuid's in `flows`). Should we add a FIXME comment or an issue or task or something to look into that later?